docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -663,6 +663,14 @@ You should see a response similar to the following:
 }
 ----
 
+The default way for a client to provide the necessary authentication to let Config Server talk to Vault is to set the X-Config-Token header.
+However, you can instead omit the header and configure the authentication in the server, by setting the same configuration properties as Spring Cloud Vault.
+The property to set is `spring.cloud.config.server.vault.authentication`.
```

Comment:
`spring.cloud.config.server.vault.authentication` should be set to one of the supported Authentication Methods. Other properties specific to the chosen authentication method may also need to be set, using the same property names as documented for `spring.cloud.vault` but using the `spring.cloud.config.server.vault` prefix.

Reply:
Good detail. Thanks. :)